src/main/cluster.ts

```diff
@@ -177,6 +177,12 @@ export class Cluster implements ClusterModel, ClusterState {
    * @observable
    */
   @observable isAdmin = false;
+  /**
+   * Does watch-api could be used for all resources, e.g. "/api/v1/services?watch=1"
```

Comment:
This comment makes no grammatical sense.

Do you mean "Is watch-api able to be used ..."

Reply:
fixed in target PR